internal/exec/engine.go

```diff
@@ -103,6 +117,39 @@ func (e Engine) Run(stageName string) error {
 	return nil
 }
 
+// isConfigCacheExist checks if we already have a cached config.
+// TODO(sohan): This is a hack to emit the journald message only
+// once for the base config (https://github.com/coreos/ignition/issues/950)
+func (e *Engine) isConfigCacheExist(cache string) (bool, error) {
```

Comment:
This function could be moved to the util package or something as e.g. `PathExists()`. But ehh, not mandatory.